src/components/executive-education-2u/ExecutiveEducation2UPage.jsx

```diff
@@ -1,22 +1,23 @@
 import React, {
-  useContext, useEffect, useMemo,
+  useContext, useEffect, useMemo, useState,
 } from 'react';
 import { Container } from '@edx/paragon';
 import { AppContext } from '@edx/frontend-platform/react';
 import { Helmet } from 'react-helmet';
 import Skeleton from 'react-loading-skeleton';
 import { logError } from '@edx/frontend-platform/logging';
-
 import NotFoundPage from '../NotFoundPage';
 import UserEnrollmentForm from './UserEnrollmentForm';
 import {
   useActiveQueryParams,
   useExecutiveEducation2UContentMetadata,
 } from './data';
+import { ExecutiveEducation2UErrorPage } from '../executive-education-2u-error';
 
 function ExecutiveEducation2UPage() {
   const { enterpriseConfig } = useContext(AppContext);
   const activeQueryParams = useActiveQueryParams();
+  const [pageError, setPageError] = useState(false);
```

Comment:
nit: [pageError, setPageError] -> [hasPageError, setHasPageError]

Reply:
Direct reference to the query used, moot point